packages/terra-tag/docs/README.md

```diff
@@ -0,0 +1,28 @@
+# Terra Tag
+
+The tag component is used for tagging and provides users a way to trigger actions in the UI. Each tag should have text.
+It can optionally display an icon along with the text.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-tag`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Tag from 'terra-tag';
+
+<Tag text="Default"
+  icon={icon}
+  theme="normal"
```

Comment:
You can remove the theme prop from the example.

Reply:
Removed it [here](https://github.com/cerner/terra-core/pull/1306/commits/8e3c99f0e06259ee8eb0b2b7e435c28cf0feca0c).